pkgs/ffigen/test/native_objc_test/util.dart

```diff
@@ -37,3 +40,42 @@ void doGC() {
   _executeInternalCommand(gcNow.cast(), nullptr);
   calloc.free(gcNow);
 }
+
+@Native<Bool Function(Pointer<Void>)>(isLeaf: true, symbol: 'isReadableMemory')
+external bool _isReadableMemory(Pointer<Void> ptr);
+
+@Native<Uint64 Function(Pointer<Void>)>(
+    isLeaf: true, symbol: 'getBlockRetainCount')
+external int _getBlockRetainCount(Pointer<Void> block);
+
+int getBlockRetainCount(Pointer<ObjCBlock> block) {
+  if (!_isReadableMemory(block.cast())) return 0;
+  if (!internal_for_testing.isValidBlock(block)) return 0;
+  return _getBlockRetainCount(block.cast());
+}
+
+@Native<Uint64 Function(Pointer<Void>)>(
+    isLeaf: true, symbol: 'getObjectRetainCount')
+external int _getObjectRetainCount(Pointer<Void> object);
+
+int getObjectRetainCount(Pointer<ObjCObject> object) {
```

Comment:
Drive by: we don't like get in Dart land:
https://dart.dev/effective-dart/design#avoid-starting-a-method-name-with-get

It seems to apply to functions in addition to methods.

Reply:
Done